solr/contrib/ltr/src/java/org/apache/solr/ltr/feature/FieldValueFeature.java

```diff
@@ -113,9 +113,10 @@ public float score() throws IOException {
             }
           }
         } catch (final IOException e) {
-          // TODO discuss about about feature failures:
-          // do we want to return a default value?
-          // do we want to fail?
+          throw new FeatureException(
```

Comment:
Not sure if we want to introduce this breaking case right now, just make a ticket for this and leave it the way it was before since we should add test cases that would cover this scenario